src/utils/string/StringUtils.js

```diff
@@ -163,6 +162,7 @@ const displayFormattedSearchLocationsV2 = (searchQuery, locationObject) => {
   return null;
 };
 
+// eslint-disable-next-line consistent-return
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Disabling eslint rules should be avoided where possible as it can lead to inconsistent code style and potential issues. If you need to disable a rule, it's good practice to include a comment explaining why.



<!-- This is an auto-generated comment by OSS CodeRabbit -->